lib/main.dart

```diff
@@ -320,6 +328,9 @@ class PendingTasksPage extends StatelessWidget {
                           content: Column(
                             mainAxisSize: MainAxisSize.min,
                             children: [
+
```

Comment:
A custom Widget to create the Text fields would make sense in my opinion, better readability and optimization would be enhanced.